subsys/bluetooth/mesh/proxy_srv.c

```diff
@@ -599,7 +599,8 @@ static int gatt_proxy_advertise(struct bt_mesh_subnet *sub)
 		}
 	}
 
-	if (sub->node_id == BT_MESH_NODE_IDENTITY_STOPPED) {
+	if (sub->node_id == BT_MESH_NODE_IDENTITY_STOPPED &&
+		bt_mesh_gatt_proxy_get() == BT_MESH_FEATURE_ENABLED) {
```

Comment:
Could you please add a reference to the spec section that specifies this as a comment?